sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -239,6 +239,7 @@ public abstract partial class PipelineTransport : System.ClientModel.Primitives.
     public partial class RequestOptions
     {
         public RequestOptions() { }
+        public bool? BufferResponse { get { throw null; } set { } }
```

Comment:
why is this nullable? Should we simply have true as the default?

Reply:
If it always has a value, I wasn't sure how to compose it with the existing value on message.BufferResponse.  

Did you have a solution to that that I didn't think of?

Reply:
What is the problem with the composition? I was thinking that the last set to message.BufferResponse wins

Reply:
the shape of the request creation helper looks like this:

```csharp
      var message = _pipeline.CreateMessage();
      var request = message.Request;
      message.BufferResponse = false;
      message.Apply(options);  // <-- sets buffer response from options
```

If options.BufferResponse defaults to true, then a client cannot specify an operation that doesn't buffer the response.

Reply:
Clients have protocol methods and convenience methods. RequestOptions are a parameter to protocol methods. Clients should not be picking whether protocol methods buffer or not. The default should be to buffer and RequestOption can change that. So I am not sure when clients "specify" that operation does not buffer.

Reply:
Discussed offline: I believe this approach -- where BufferResponse is nullable -- will work without changes to the generator today.   @KrzysztofCwalina is concerned that a tri-state property complicates things for only a small benefit.

We agreed that, given short timeline needs for this feature, we can merge this as-is to unblock unbranded client development and hopefully return to revisit to understand what generator changes would be required to support the non-nullable bool property, with the story: "for protocol methods, we buffer by default. If the user does not like it, they can change the option. Convenience methods set the option however they want."